### DISCUSSION TITLE
Problem with using ContentPicker on RazorPages

### DISCUSSION THREAD

**Discussion by Diman96** [OrchardCMS/OrchardCore#8135] [Q&A]
I can't use `ContentPicker` on `RazorPages`, I did not find any information in the documentation other than this https://docs.orchardcore.net/en/dev/docs/reference/modules/ContentFields/#contentpicker-field-example. 
My `_ViewImports`
![image](https://user-images.githubusercontent.com/12182810/103016607-f8dc0f80-454a-11eb-8f67-8b1947dce51e.png)
How i try use `ContentPicker` with `@inject OrchardCore.IOrchardHelper Orchard` on the top of page
![image](https://user-images.githubusercontent.com/12182810/103016738-35a80680-454b-11eb-9fe7-5eac9400f321.png)

**Comment by hishamco** [comment]
According to the docs you could use:

```html
@foreach (var contentItem in await Orchard.GetContentItemsByIdAsync(Model.ContentItemIds))
{
    @contentItem.DisplayText
}
```

Seems the issue in your above example is the content item Ids
